handler/EventHandler.go

```diff
@@ -0,0 +1,209 @@
+package handler
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/ONSdigital/dp-import-reporter/config"
+
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/coocood/freecache"
+)
+
+var failed = "failed"
+
+//main function that triggers everything else
```

Comment:
There are some standards around function comments in Go - https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Comments on exported methods should start with the method name